client/signup/step-wrapper/index.jsx

```diff
@@ -195,6 +200,11 @@ class StepWrapper extends Component {
 								subHeaderText={ this.subHeaderText() }
 								align={ align }
 							/>
+							{ headerImageUrl && (
+								<div className="step-wrapper__image">
```

Comment:
`step-wrapper__header-image` might be better to avoid some naming conflicts in the future. But it's also good to keep using the current one.

Reply:
good point